### ISSUE TITLE
`test_minimum_service_factor` marked as xfail

### ISSUE THREAD

**Issue by tsmbland** [EnergySystemsModellingLab/MUSE_OS#400]
Once #335 is sorted, this test should pass, so we must remember to remove the xfail mark

**Comment by HarmonicReflux** [comment]
I think this is related to earlier issue #359

**Comment by dalonsoa** [comment]
No, it is not. This is something that @tsmbland marked the other day as `xfail` because of a known cause that will be sorted with #335

**Comment by dalonsoa** [comment]
@HarmonicReflux , see https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/381#issuecomment-2206788417
